spring-cloud-starter-stream-source-http/README.adoc

```diff
@@ -33,11 +33,11 @@ The **$$http$$** $$source$$ supports the following configuration properties:
 $$http.cors.allow-credentials$$:: $$Whether the browser should include any cookies associated with the domain of the request being annotated.$$ *($$Boolean$$, default: `$$<none>$$`)*
 $$http.cors.allowed-headers$$:: $$List of request headers that can be used during the actual request.$$ *($$String[]$$, default: `$$<none>$$`)*
 $$http.cors.allowed-origins$$:: $$List of allowed origins, e.g. "http://domain1.com".$$ *($$String[]$$, default: `$$<none>$$`)*
-$$http.enable-csrf$$:: $$The security CSRF enabling flag. Makes sense only if 'enableSecurity = true'.$$ *($$Boolean$$, default: `$$false$$`)*
-$$http.enable-security$$:: $$The security enabling flag.$$ *($$Boolean$$, default: `$$false$$`)*
 $$http.mapped-request-headers$$:: $$Headers that will be mapped.$$ *($$String[]$$, default: `$$<none>$$`)*
 $$http.path-pattern$$:: $$An Ant-Style pattern to determine which http requests will be captured.$$ *($$String$$, default: `$$/$$`)*
 $$server.port$$:: $$Server HTTP port.$$ *($$Integer$$, default: `$$8080$$`)*
+$$spring.cloud.stream.security.csrf-enabled$$:: $$The security CSRF enabling flag. Makes sense only if security 'enabled` is `true'.$$ *($$Boolean$$, default: `$$true$$`)*
+$$spring.cloud.stream.security.enabled$$:: $$The security enabling flag.$$ *($$Boolean$$, default: `$$false$$`)*
```

Comment:
Why do you include these properties here explicitly?
I don't think that it is a responsibility of this HTTP app to expose them.
Or are you going to say the same in all the other apps starters?

Thanks

Reply:
Currently the `app-starters-security-common for security` is applied only to the `http-source` app stater. 
Since i've removed the analogous http security properties it though it might be useful to list the new one,  although they come form an external module. 

What do you think?  I can remove it if you think that is inconsistent with other porops policies?

Reply:
Yes, please, remove it. 
Since we have now a general solution which covers this HTTP Source as well, we don't need to duplicate a logic and docs here.
Everything is covered from that common module perspective.

As I said: we just need to have a proper docs in that module, plus on the App Starters level: https://docs.spring.io/spring-cloud-stream-app-starters/docs/Darwin.SR2/reference/htmlsingle/

Reply:
Ok done